back/src/routes/airtable.ts

```diff
@@ -42,11 +43,17 @@ airtableRouter.post('/application_status', (req, res, next) => {
       status: "",
       description: "",
     };
+  
+
+    if(req.body.firstName == undefined || req.body.firstName == "" || req.body.lastName == undefined || req.body.lastName == ""){
+      res.send(404);
+      return;
+    }
     base('2021 Form Responses').select({
       fields: ["Applicant First Name", "Applicant Last Name", "PWA Status", "PWA Status Description"],
       filterByFormula: `AND({Applicant First Name} = '${req.body.firstName}', {Applicant Last Name} = '${req.body.lastName}')`
     }).firstPage(function(err, records) {
-      if(err) { console.error(err); return; }
```

Comment:
Isn't this return necessary?

Reply:
Yes, I don't think I meant to delete it.